lua/damagelogs/shared/lang/deutsch.lua

```diff
@@ -393,5 +394,15 @@ DamagelogLang.deutsch = {
     -- Discord webhook
     webhook_AdminsOnline = "Es sind Admins online.",
     webhook_NoAdminsOnline = "Es sind keine Admins online.",
-    webhook_ServerInfo = "Aktuelle Karte: %s, Runde: %s"
+    webhook_ServerInfo = "Aktuelle Karte: %s, Runde: %s",
+    webhook_report_forgiven_or_kept = "Vergeben oder Behalten",
+    webhook_report_forgiven = "Vergeben",
+    webhook_report_kept = "Behalten",
+    webhook_report_finished_by = "Report bearbeitet von",
+    webhook_report_finished_time_taken = "Gebrauchte Zeit",
+    webhook_report_finished_conclusion = "Schlussfolgerung",
+    webhook_header_report_submitted = "Ein Report wurde eingereicht (ID %s)",
+    webhook_header_report_forgiven = "Report vergeben (ID %s)",
+    webhook_header_report_kept = "Report behalten (ID %s)",
+    webhook_header_report_finished = "Report beendet (ID %s)"
```

Comment:
Thanks a lot for adding/fixing these!

Reply:
Yeah np, let me know if I can help you with more translations in this or future projects 💪